__tests__/commands/install/integration.js

```diff
@@ -236,6 +236,13 @@ test.concurrent('install file: protocol without cache', async (): Promise<void>
   });
 });
 
+test.concurrent('install file: local packages with local dependencies', async (): Promise<void> => {
+  await runInstall({}, 'install-file-local-dependency', async (config, reporter) => {
+    const reinstall = new Install({}, config, reporter, await Lockfile.fromDirectory(config.cwd));
+    await reinstall.init();
```

Comment:
What would the expected state be here? I'm fine with adding that, I just want to make sure I'm asserting the right thing. At the moment, I'm relying on the exception being thrown to fail the test. I could catch the exception and set a boolean and assert on that boolean or something similar.